pytest-embedded/pytest_embedded/unity.py

```diff
@@ -291,9 +292,13 @@ def merge(self, junit_files: List[str]):
                 raise ValueError(f'Could\'t find test case {test_case_name}, dumped into "debug.xml" for debugging')
 
             junit_case_is_fail = junit_case.find('failure') is not None
-            junit_parent.remove(junit_case)
+
+            junit_case.attrib['PYTHON_FUNC'] = '1'
```

Comment:
how about rename it to `is_unity_case`?

developers may get confused when they see this, but they're not having any unity test cases